azure-pipelines.yaml

```diff
@@ -70,6 +70,9 @@ parameters:
       - deployment: "apim"
         environment: "dev"
         dependsOn: "Precheck"
+      - deployment: "apim"
```

Comment:
you've updated the test file for the other one but this is staging?

Reply:
my bad - clicked wrong folder, should be stg

Reply:
added to `stg.tfvars` now